### PR TITLE
Feat(eos_cli_config_gen): Support additional options for IPv6 ND under SVIs

### LINKED ISSUE #3933: Feat(eos_cli_config_gen): Support additional options for IPv6 ND under SVIs
### Enhancement summary

Under SVIs, add the options for:

```
ipv6 nd other-config-flag
ipv6 nd cache expire ...
```

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

If you have IPv6 and use dynamic options for providing an IPv6 address you might need "ipv6 nd other-config-flag". Also it might be useful to set the cache expire which is more or less equivalent to the ARP timeout in IPv4.

### Describe the solution you would like

adding additional options for SVIs

### Describe alternatives you have considered

raw_eos_cli is working but not the preferred way

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by bjmeuer**
## Change Summary

Under SVIs, added the options for:

ipv6 nd other-config-flag
ipv6 nd cache ...

## Related Issue(s)

Fixes #3933 

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
for the SVI the following was added:

```
        ipv6_nd_other_config_flag:
          type: bool
          description: Set the "other stateful configuration" flag in IPv6 router advertisements.
        ipv6_nd_cache:
          type: dict
          description: IPv6 neighbor cache options.
          keys:
            dynamic_capacity:
              type: int
              description: Capacity of dynamic cache entries.
              convert_types:
                - str
              min: 0
              max: 4294967295
            expire:
              type: int
              description: Cache entries expirery in seconds.
              convert_types:
                - str
              min: 1
              max: 65535
            refresh_always:
              type: bool
              description: Force refresh on cache expiry.
```

## How to test

```
tenants:
  - name: BLA
    vlan_aware_bundle_number_base: 300
    mac_vrf_vni_base: 20000
    vrfs:
      - name: BLA_VRF
        vrf_vni: 789654
        svis:
          - id: 4088
            name: SVI_1
            enabled: true
            evpn_vlan_bundle: bundle3
            ip_address: 1.1.1.1/24
            ipv6_address_virtuals: [ 2002:a:1::22/64 ]
            structured_config:
              ipv6_nd_other_config_flag: true
              ipv6_nd_cache:
                dynamic_capacity: 900
                expire: 250
                refresh_always: true
```

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Only needs a rerun of molecule `eos_cli_config_gen_deprecated_vars` to be ready.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md [hunk 1]
```diff
@@ -409,6 +412,15 @@ interface Vlan336
    ipv6 enable
    ipv6 address 2a00:3::1/64
 !
+interface Vlan339
+   description v6 nd options
+   ipv6 nd cache expire 250
+   ipv6 nd cache dynamic capacity 900
+   ipv6 nd cache refresh always
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md:434]: This command `ipv6 nd cache refresh always` shows me invalid on EOS 4.32. Can you please confirm it this is platform or EOS version dependent

Comment (bjmeuer) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md:434]: @MaheshGSLAB Thanks for having a look, let me double check if this is something platform dependent, but at least for me it is fine in 4.32.1F version:

```
B1-BR-0001(config-if-Vl10)#ipv6 nd cache refresh ?
  always  Force refresh on cache expiry

B1-BR-0001(config-if-Vl10)#ipv6 nd cache refresh always ?
  <cr>

B1-BR-0001(config-if-Vl10)#sh vers
Arista cEOSLab
Hardware version:
Serial number: 7ABC871EEEA686DFEFD7E4119B37E515
Hardware MAC address: 001c.73f6.f48b
System MAC address: 001c.73f6.f48b

Software image version: 4.32.1F
```

which switch model did you test with?

Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md:434]: i checked on 4.32.0F, so yes there is EOS version dependency.

**Review by MaheshGSLAB — APPROVED** [review]
LGTM, tested on lab

**Review by sugetha24 — APPROVED** [review]

**Pull request merged**